templates/history.html

```diff
@@ -12,15 +12,15 @@
     <h1>Previous Workouts</h1>
     <table>
         <tr>
-            <th>Timestamp (under construction)</th>
+            <th>Date</th>
             <th>Duration (minutes)</th>
             <th>Weight (kilos) </th>
             <th>Exercise Type</th>
             <th>Calories burned (kcal)</th>
         </tr>
         {% for i in range(num_workouts) %}
         <tr>
-            <td>Under construction</td>

```

Comment:
will be constructing soon!

Reply:
I did that, it just showed up as a deletion and a new insertion.